docs/source/manual/core.rst

```diff
@@ -1219,14 +1219,23 @@ and then registering the exception mapper:
 Overriding Default Exception Mappers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you want more control, you can disable the exception mappers Dropwizard provides by default. This is done
-by setting ``server.registerDefaultExceptionMappers`` to ``false``. Since this disables all default exception
-mappers make sure to re-enable exception mappers that are wanted. The default exception mappers are:
-
-- ``LoggingExceptionMapper<Throwable>``
-- ``JerseyViolationExceptionMapper``
-- ``JsonProcessingExceptionMapper``
-- ``EarlyEofExceptionMapper``
+To override a specific exception mapper, register your own class that implements the same
+``ExceptionMapper<T>`` as one of the default. For instance, we can customize responses caused by
```

Comment:
Yeah, couple sentences later

> See the class ``ExceptionMapperBinder`` for a list of the default exception mappers.

Which is similar to the other snippets in the documentation.